Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM python:3.7.0
+FROM python:3.11.2
 
 WORKDIR /usr/src/
 
-RUN apt-get update && apt-get -y install jq
```

Comment:
I wouldn't get rid of this, there are always security updates to add, and jq is always handy.